### PR TITLE
Templated dashboards for ECS services v1

### PR DESCRIPTION
**Pull request by peggyl**
Create automated dashboards for env/apps in ECS.

Example on latest build: 
https://peggy--grafana-service.int.clever.com:83/dashboard/script/scripted_services.js?env=peggy&app=grafana-service

### CODE REVIEW

**Comment by peggyl** [comment]
We can probably do more/better JS refactoring in a later iteration, but I'd like to ship something usable by EOW in preparation for Monday.

I think @johnhuangclever has used this a couple of times for load testing metrics. I can definitely prioritize bug fixes or medium/major issues uncovered.

File: dashboards/scripted_services.js [hunk 1]
```diff
@@ -0,0 +1,408 @@
+/* global _ */
+
+/*
+ * Dashboard for vieiwng metrics on ECS services
```
Comment (nathanleiby) [inline dashboards/scripted_services.js]: nit: s/vieiwng/viewing

Comment (peggyl) [inline dashboards/scripted_services.js]: Nice catch! Fixed and updated another outdated comment.

**Comment by nathanleiby** [comment]
lgtm!

**Comment by peggyl** [comment]
I'm going to go ahead and merge/deploy this, then add in some fixes and improvements.

**Pull request merged**